docs/src/components/DocLayout.tsx

```diff
@@ -27,10 +28,14 @@ const StyledWrapper = styled.div`
 `;
 
 const StyledMain = styled.main`
+  position: relative;
   display: flex;
   flex-direction: column;
   overflow: hidden;
-  padding: 2rem;
+  padding: 2rem ${CONTENT_PADDING};
```

Comment:
Because `CONTENT_PADDING` refers only to horizontal padding, but it seemed verbose to name it that way. I wanted to keep the fact that vertical and horizontal padding are the same incidental.

Reply:
I know naming is hard, but I definitely didn't understand that from the name. Do you anticipate the vertical and horizontal padding being different in the future?

Reply:
Now I use it for both paddings. ✅